website/_dogfooding/_pages tests/analytics.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Layout from '@theme/Layout';
+
+// See https://github.com/facebook/docusaurus/issues/6337#issuecomment-1012913647
+export default function Analytics(): JSX.Element {
+  return (
+    <Layout>
+      <h1>Test Analytics</h1>
+      <div>
+        <button
+          type="button"
+          onClick={() => {
+            window.gtag('event', 'docusaurus-test-event', {
```

Comment:
side note: it doesn't work in dev/deploy-preview because we don't load gtag script there

We should probably expose a `gtag | undefined` instead of just `gtag`, to make it clear that user has to program defensively against this api